mnemonic-core/src/test/java/org/apache/mnemonic/DurablePersonNGTest.java

```diff
@@ -91,6 +104,7 @@ public boolean reclaim(Long mres, Long sz) {
           mother = PersonFactory.create(act);
           mother.setAge((short) (50 + rand.nextInt(50)));
           mother.setName(String.format("Name: [%s]", UUID.randomUUID().toString()), true);
+          mother.setNamebuffer(mbh, true);
```

Comment:
Please create & init a new buffer for this person, each person has their own buffer and do not share buffer between different person.